app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -20,6 +23,48 @@ public int layout() {
 
     @Override
     public void init() {
+        LoginPresenter loginPresenter = new LoginPresenter();
```

Comment:
Remember we don't instantiate the `Presenter`! It's received injected inside the `WolmoFragment`.